### PR TITLE
Fix expressions replacing inside introduce refactorings

### PR DESCRIPTION
**Pull request by ortem**
1. Replacing `RsExpr` with `PsiIdentifier` leads to problems with parsing result, so now it uses `createExpression`.
2. `MemberInplaceRenamer` may not work correctly when the user starts typing right after introducing a parameter. So it was replaced with `RustInPlaceVariableIntroducer` (also providing all suggested names).

### CODE REVIEW

File: src/main/kotlin/org/rust/ide/refactoring/introduceVariable/impl.kt [hunk 1]
```diff
@@ -134,7 +133,7 @@ private fun findAnchor(expr: PsiElement): PsiElement? {
     return anchor
 }
 
-private class RustInPlaceVariableIntroducer(
+class RustInPlaceVariableIntroducer(
```
Comment (vlad20012) [inline src/main/kotlin/org/rust/ide/refactoring/introduceVariable/impl.kt]: Maybe move the class to some common pace?

Comment (ortem) [reply src/main/kotlin/org/rust/ide/refactoring/introduceVariable/impl.kt]: Done, created `introduceUtils.kt` for such needs.

File: src/main/kotlin/org/rust/ide/refactoring/introduceUtils.kt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+package org.rust.ide.refactoring
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiNamedElement
+import com.intellij.refactoring.introduce.inplace.InplaceVariableIntroducer
+
+class RustInPlaceVariableIntroducer(
```
Comment (Undin) [inline src/main/kotlin/org/rust/ide/refactoring/introduceUtils.kt]: Let's use `Rs` prefix for such names to be consistent with all other code
And maybe it's worth to rename the file to `RsInPlaceVariableIntroducer`? It contains only this class

**Comment by Undin** [comment]
bors retry

**Comment by Undin** [comment]
bors r=vlad20012

**Pull request merged**